docs/tooltip.md

```diff
@@ -0,0 +1,159 @@
+---
+id: tooltip
+title: Tooltip
+---
+
+Easy to use and customisable tooltip.
+
+<img alt="tooltip example gif" width='290' src='/react-native-elements/img/tooltipExample.gif'>
+
+### Usage.
+
+```js
+import { Tooltip, Text } from 'react-native-elements';
+
+...
+
+<Tooltip popover={<Text>Info here</Text>}>
+  <Text>Press me</Text>
+</Tooltip>
+
+```
+
+### Props
+
+* [`backgroundColor`](#backgroundcolor)
+* [`containerStyle`](#containerStyle)
+* [`height`](#height)
+* [`highlightColor`](#highlightColor)
+* [`onClose`](#onClose)
+* [`onOpen`](#onOpen)
+* [`pointerColor`](#pointerColor)
+* [`popover`](#popover)
+* [`toggleOnPress`](#toggleOnPress)
+* [`width`](#width)
+* [`withOverlay`](#withOverlay)
+* [`withPointer`](#withPointer)
+
+---
+
+# Reference
+
+### `backgroundColor`
+
+sets backgroundColor of the tooltip and pointer.
+
+|  Type  | Default |
+| :----: | :-----: |
+| string | #617080 |
+
+---
+
+### `containerStyle`
+
+Passes style object to tooltip container
+
+|  Type   | Default  |
+| :-----: | :------: |
+| Object | {} |
```

Comment:
For any types that are an object, let's stick to this convention. See https://github.com/react-native-training/react-native-elements/blame/master/docs/listitem.md#L307 for example

Reply:
Also the containerStyle in the source technically has other methods on it
```js
position: 'absolute',
left: x,
top: y,
width,
height,
backgroundColor,
// default styles
display: 'flex',
alignItems: 'center',
justifyContent: 'center',
flex: 1,
borderRadius: 10,
padding: 10,
```

So you could use `inherited styling` as the value here.